unit-tests/src/org/commcare/android/tests/caselist/EntityListCalloutDataTest.java

```diff
@@ -112,9 +115,7 @@ public void testCalloutResultWithNoColumnTest() {
 
     private void performFingerprintCallout() {
         // make entity list callout to 'fingerprint identification'
-        ImageButton calloutButton =
-                (ImageButton)entitySelectActivity.findViewById(R.id.barcodeButton);
-        calloutButton.performClick();
+        entitySelectActivity.barcodeScanOnClickListener.onClick(null);
```

Comment:
Yes. This test was previously using the version of the callout button that is only visible on old devices. By changing the `setupPreHoneycombFooter()` method to only init those views when they are actually being used, I rendered the R.id.barcodeButton unusable in this test, since it is never initialized. The new implementation is a version-agnostic way to test this, since both UI implementations use the same onClickListener